### ISSUE TITLE
GUI: Scripts which catch keyboard interrupt with inputs should work

### ISSUE THREAD

**Issue by John-Holt-Tessella** [ISISComputingGroup/IBEX#5392]
As a developer I want to write a script such that I catch the keyboard interrupt (ctrl + c) and then can offer an option to the user. Currently this does not work because either the input is ignored or the prompt is not returned to the user. 

Test script:

```
try:
    while True: pass
except KeyboardInterrupt:
    import time
    time.sleep(0.2)  # comment line for different effect
    a = input("Type something")
    print("You wrote {}".format(a))
```

If sleep line is in then prompt is not returned, if it is not then input is not taken.

### Acceptance Criteria

- Script runs correctly (having a sleep in is acceptable)
- Reflectometry script is removed (ctrl + c line 237 in scripts `technique/reflectometry/base.py`)

**Comment by DominicOram** [comment]
Is this related to https://github.com/ISISComputingGroup/IBEX/issues/5160?

**Comment by John-Holt-Tessella** [comment]
Yes but it is different, this is not ctrl +c in the middle of an input.
